README.md

```diff
@@ -12,6 +12,18 @@ Supports L2 distance, inner product, and cosine distance
 
 [![Build Status](https://github.com/ankane/pgvector/workflows/build/badge.svg?branch=master)](https://github.com/ankane/pgvector/actions)
 
+## Docker
```

Comment:
Let's put this under the Additional Installation Methods section (added a few days ago)